frontend/src/views/AppView.vue

```diff
@@ -118,6 +119,19 @@ const state = reactive<LocalState>({
   rentYear: 1,
 });
 
+const title = computed(() => {
+  const curRoute = router.currentRoute.value;
+  if (curRoute.name === "provider") {
+    if (curRoute.params.provider === "AWS") {
+      return `The Ultimate AWS RDS Instance Pricing Sheet`;
```

Comment:
We will probably change this later again. e.g. after adding database spec dimension, it will be

AWS RDS db.m4.large Instance Pricing Sheet